shell/platform/darwin/ios/framework/Source/SemanticsObject.h

```diff
@@ -231,7 +231,7 @@ constexpr float kScrollExtentMaxForInf = 1000;
                                  bridge:(fml::WeakPtr<flutter::AccessibilityBridgeIos>)bridge
     NS_DESIGNATED_INITIALIZER;
 
-@property(nonatomic, weak) SemanticsObject* semanticsObject;
+@property(nonatomic, assign) SemanticsObject* semanticsObject;
```

Comment:
Made this `assign` to get the property synthesis.  I'm going to change it back to `weak` when I do the ARC migration, which allows weak property synthesis.  This header isn't public.

Reply:
oh wow i didn't know they have `weak` support in MRC. 

> Made this assign to get the property synthesis
So `weak` in MRC doesn't synthesize the getter/setter/ivar?